community-membership.md

```diff
@@ -0,0 +1,168 @@
+    - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+    - Filing or commenting on issues on GitHub
+    - Contributing to a project, or community discussions (e.g. meetings, Slack, email discussion
+      forums, Stack Overflow)
+- Subscribed to https://groups.google.com/g/sigstore-dev
+- Actively contributing to 1 or more projects.
+- Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
+    - Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating
+      on issues, etc.
+    - Sponsors must be reviewers or approvers in at least one CODEOWNERS file.
+    - Sponsors must be from multiple member companies to demonstrate integration across community.
+- **[Open an issue][membership request] against the sigstore/community repo**
+   - Ensure your sponsors are @mentioned on the issue
+   - Complete every item on the checklist ([preview the current version of the template][membership template])
+   - Make sure that the list of contributions included is representative of your work on the project.
+- Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
+
+### Responsibilities and privileges
+
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- They can be assigned to issues and PRs, and people can ask members for reviews with a `/cc @username`.
+- Tests can be run against their PRs automatically.
+
+**Note:** members who frequently contribute code are expected to proactively
+perform code reviews and work towards becoming a primary *reviewer* for the
+subproject that they are active in.
+
+## Reviewer
+
+Reviewers are able to review code for quality and correctness on some part of a
+subproject. They are knowledgeable about both the codebase and software
+engineering principles.
+
+**Defined by:** *reviewers* entry in an CODEOWNERS file in a repo owned by the
+Sigstore project.
+
+Reviewer status is scoped to a part of the codebase.
+
+**Note:** Acceptance of code contributions requires at least one approver in
+addition to the assigned reviewers.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be a reviewer in
+an [CODEOWNERS] file (for repos using the bot).
+
+- member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- Sponsored by a project approver
+  - With no objections from other approvers
+  - Done through PR to update the OWNERS file
+- May either self-nominate, be nominated by an approver in this subproject, or be nominated by a robot
```

Comment:
Nominated by a robot? Do we want automation for this?

Reply:
clean up